.github/workflows/cd.yml

```diff
@@ -0,0 +1,25 @@
+name: CD
+on:
+  push:
+    branches: [master]
+jobs:
+  CD:
+    runs-on: ubuntu-latest
+    name: CD
+    environment: Release

```

Comment:
'Concurrent deployment' is managed using the wait timer of the environment. This results in : if in 15' no commit comes in, then I release.